ui/tests/e2e/login.test.ts

```diff
@@ -0,0 +1,76 @@
+import puppeteer from 'puppeteer'
+
+// const LOGIN_URL = 'http://localhost:3000/#/signin'
+const LOGIN_URL = process.env.CI
+  ? 'http://127.0.0.1:12333/dashboard/#/signin'
+  : 'http://localhost:3000/#/signin'
+
+describe('Login', () => {
+  let browser
+  beforeAll(async () => {
+    // browser = await puppeteer.launch({
+    //   headless: false,
+    //   slowMo: 100,
+    // })
+    browser = await puppeteer.launch()
+  })
+
+  afterAll(() => {
+    browser.close()
+  })
+
+  it(
+    'should login success by correct password',
+    async () => {
+      const page = await browser.newPage()
+
+      await page.goto(LOGIN_URL)
+
+      const title = await page.title()
+      expect(title).toBe('TiDB Dashboard')
+
+      await page.waitForSelector('button[type=submit]')
```

Comment:
Looks like we'd better assign an ID (or a customized attribute) to pinpoint those e2e interact targets, to avoid accidentally selecting others

Reply:
```js
const buttonEl = await page.waitForSelector(..)
await buttonEl.click()
```